shared/chat/conversation/messages/wrapper/wrapper-author/container.js

```diff
@@ -12,8 +12,8 @@ const mapStateToProps = (state: TypedState, {message, previous, innerClass, isEd
   const isYou = state.config.username === message.author
   const isFollowing = state.config.following.has(message.author)
   const isBroken = state.users.infoMap.getIn([message.author, 'broken'], false)
-  const lastPositionOrdinal = state.chat2.orangeLineMap.get(message.conversationIDKey)
```

Comment:
I'm not super familiar with the difference between ordinals and message ids, but it seems like if we're only setting the orange line on the load of an inbox then we don't need to use our own ordinals and can just use message ids provided. Definitely could be wrong though, please lmk